database/User.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose')
+mongoose.connect('mongodb://localhost/lizardboard')
+const db= mongoose.connection
+
+const UserSchema = require('./UserSchema')
+db.on('error', console.error.bind(console,'connection error:'))
```

Comment:
We need to separate the setup of the database connection from the definition of the models...